.github/templates/model_regression_test_results.tmpl

```diff
@@ -136,18 +139,21 @@ The print_result_core template returns data depends on available fields.
 `{{ $config_data.train_run_time }}`| `{{ $config_data.test_run_time }}`|
 {{ end }}
 {{- end}}
+{{- end}}
 {{- end -}}
 {{- end -}}
 
 {{- $results_main := (datasource "results_main") -}}
-{{ range $dataset, $config := (datasource "data")}}
-{{ $dataset_main := (index $results_main $dataset) -}}
-{{- if (index $config (index (keys $config) 0)).external_dataset_repository -}}
-Dataset: `{{$dataset}}`, Dataset repository branch: `{{ (index $config (index (keys $config) 0)).dataset_repository_branch }}` (external repository), commit: `{{ (index $config (index (keys $config) 0)).dataset_commit }}`
-Configuration repository branch: `{{ (index $config (index (keys $config) 0)).config_repository_branch }}`
+{{ range $dataset, $results_for_dataset := (datasource "data")}}
+{{ $results_for_dataset_main := (index $results_main $dataset) -}}
+{{ $content_dicts := index $results_for_dataset (index (keys $results_for_dataset) 0) -}}
```

Comment:
The array `$content_dicts` contains multiple dicts (a.ka. maps), but they happen to each have the same attributes:
- dataset_commit
- dataset_repository_branch

Therefore we can take the first element.
This could probably be cleaner, I only started with a slight refactoring about renaming things here

Reply:
Ahh, no that makes sense, I think I was just looking for some sort of summary function. But now that I look at it some more and the test, I should expect one result per row... right? 🤞